server/middleware/prepaids.coffee

```diff
@@ -145,6 +234,7 @@ module.exports =
     prepaids = yield Prepaid.find(query, {creator: 1, startDate: 1, endDate: 1, maxRedeemers: 1, redeemers: 1}).lean()
     console.log new Date().toISOString(), 'prepaids', prepaids.length
     teacherIds = []
+    # NOTE: May not correctly account for shared licenses
```

Comment:
This should be fine. Checked `AdminClassroomsProgressView`, it's just for owner lookups.